### ISSUE TITLE
Unable to Respec MC or Other Characters

### ISSUE THREAD

**Issue by Jim173** [BarleyFlour/RespecMod#178]
Since the update today, I am unable to do a respec on any characters. I did try to uninstall and reinstall, but no luck. It will let me select everything for level 1 on whichever character, it simply fails to change anything.

**Comment by mlshort** [comment]
I am having the same exact issue.

**Comment by JaksIRL** [comment]
Yes it seems the update broke it. When I respec a companion beginning at level 1 it behaves normally, but when I  submit it, it appears that nothing changes. I wish they would stop pushing these stupid updates to this game every 2 hours that changes nothing and breaks all the mods.

**Comment by Putger** [comment]
Same issue here since last patch. No idea where it goes wrong but good to see it's not on me.

**Comment by JaksIRL** [comment]
> 
Wow bro already fixed? You are a life saver, and fast!

**Comment by mlshort** [comment]
I am confused about the 'fixed' comment above.  On review of the repository, cannot see anything changed.

On review of the current source code, while not a C# dev at all, I would suggest some added logging for failures when encountered in this mod.

In the meantime, I have gone on to use the respec feature in the 'ToyBox' mod, which seems to work OK for my immediate needs.

**Comment by JaksIRL** [comment]
> I am confused about the 'fixed' comment above. On review of the repository, cannot see anything changed.
> 
> On review of the current source code, while not a C# dev at all, I would suggest some added logging for failures when encountered in this mod.
> 
> In the meantime, I have gone on to use the respec feature in the 'ToyBox' mod, which seems to work OK for my immediate needs.

A new release was put out this morning. I am looking right at it. IDK why you can't see it.

**Comment by mlshort** [comment]
Thank you for your fast reply.

I am looking at the diffs for version v1.10.11 (which I presume is the most current version).

The only changes I see checked into the repo are:

      {"Id": "RespecWrath","Version": "v1.10.10"}
        {"Id": "RespecWrath","Version": "v1.10.11"}

So, from my end, only the change in the version number got checked into the repo, which would not fix anything.  :)

**Comment by JaksIRL** [comment]
> I don't know what to tell  you, my dude. Mod manager said there was a new version this morning, I came here and downloaded the one that was released this morning, and it fixed my issue with the updated game.

**Comment by mlshort** [comment]
Yes, there is a new version posted and a new binary (.DLL).

However, it does not appear any meaningful source code changes were checked in with the latest binary.

As it is, it is impossible to see what actually changed from v1.10.10 to v1.10.11.

You can check here and see:
https://github.com/BarleyFlour/RespecMod/releases/tag/v1.10.11

This does not make me feel comfortable installing or using.
